routes/routes.js

```diff
@@ -25,10 +25,13 @@ module.exports = function(app, passport) {
     })
   })
 
-  router.post('/create', function(req,res, next){
+  router.post('/:gameid/:userid', isLoggedIn, function(red,res,next){
 
-  });
+    let game = req.params.gameid;
+    let user = req.params.userid;
 
+    actions.addPlayer(game, user);
```

Comment:
You can shorten this by just putting `actions.addPlayer(req.params.gameid, req.params.userid)`